app/controllers/opendata/agents/nodes/dataset/resource_controller.rb

```diff
@@ -6,36 +6,36 @@ class Opendata::Agents::Nodes::Dataset::ResourceController < ApplicationControll
   before_action :set_dataset
 
   private
-    def set_dataset
-      @dataset_path = @cur_path.sub(/\/resource\/.*/, ".html")
+  def set_dataset
+    @dataset_path = @cur_path.sub(/\/resource\/.*/, ".html")
 
-      @dataset = Opendata::Dataset.site(@cur_site).and_public.
-        filename(@dataset_path).
-        first
+    @dataset = Opendata::Dataset.site(@cur_site).and_public.
+      filename(@dataset_path).
+      first
 
-      raise "404" unless @dataset
-    end
+    raise "404" unless @dataset
+  end
 
   public
-    def index
-      redirect_to @dataset_path
-    end
+  def index
+    redirect_to @dataset_path
+  end
 
-    def download
-      @item = @dataset.resources.find_by id: params[:id], filename: params[:filename].force_encoding("utf-8")
-      @item.dataset.inc downloaded: 1
+  def download
+    @item = @dataset.resources.find_by id: params[:id], filename: params[:filename].force_encoding("utf-8")
+    @item.dataset.inc downloaded: 1
 
-      @cur_node.layout_id = nil
-      send_file @item.file.path, type: @item.content_type, filename: @item.filename,
-        disposition: :attachment, x_sendfile: true
-    end
+    @cur_node.layout_id = nil
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```

Comment:
__[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

<!-- Comment by [SideCI](https://sideci.com) -->